code/game/machinery/suit_storage_unit.dm

```diff
@@ -295,6 +295,7 @@
 		src,
 		choices,
 		custom_check = CALLBACK(src, PROC_REF(check_interactable), user),
+		require_near = !issilicon(user),
```

Comment:
Use the define `issiliconoradminghost` so that admin ghosts can use it too.

Reply:
added in 673a2d1